src/dosext/builtins/lredir.c

```diff
@@ -170,14 +168,18 @@ ShowMyRedirections(void)
         driveCount++;
         printf("%-2s = %-20s ", deviceStr, resourceStr);
 
-        /* read attribute is returned in the device parameter */
+        /* read attribute is returned in the device options */
         printf("attrib = ");
-        if (deviceParam > 1)
-          printf("CDROM:%i, ", deviceParam >> 1);
-        if (deviceParam & READ_ONLY_DRIVE_ATTRIBUTE)
+        if (deviceOptions & 6)
```

Comment:
I think that should be 14 (not 6), as mfs.c allows
for 4 cdroms, and 0 for none.

Reply:
Okay, but I think I'm going to have to drop this PR for now anyway as I ran out of time.